addons/sale_stock/tests/test_create_perf.py

```diff
@@ -97,7 +97,7 @@ def test_light_sales_orders_batch_creation_perf_without_taxes(self):
         # + 2 SQL insert
         # + 2 queries to get analytic default tags
         # + 9 follower queries ?
-        with self.assertQueryCount(admin=49):  # com 46
+        with self.assertQueryCount(admin=50):  # com 46
```

Comment:
The difference comes from `base_tags |= product.sudo().account_tag_ids`. Before, in case of no-tax, we didn't fetch the product's tags and now, the method computing the taxes fetche the product's tags in any case